modules/registry-configuring-storage-openstack-user-infra.adoc

```diff
@@ -6,14 +6,14 @@
 = Registry storage for {rh-openstack} with user-provisioned infrastructure
 
 You must set up the storage medium manually and configure the settings in the
-registry CRD.
+registry Custom Resource Definition (CRD).
```

Comment:
As near as I can tell, these are all wrong.

Reply:
It's this:

```
oc describe configs.imageregistry.operator.openshift.io cluster
Name:         cluster
Namespace:
Labels:       <none>
Annotations:  <none>
API Version:  imageregistry.operator.openshift.io/v1
Kind:         Config
Metadata:
  Creation Timestamp:  2020-07-12T05:38:52Z
  Finalizers:
    imageregistry.operator.openshift.io/finalizer
  Generation:  4
  Managed Fields:
    API Version:  imageregistry.operator.openshift.io/v1
    Fields Type:  FieldsV1
    fieldsV1:
      f:spec:
        f:defaultRoute:
        f:storage:
          f:pvc:
            .:
            f:claim:
    Manager:      oc
    Operation:    Update
    Time:         2020-07-12T05:58:20Z
    API Version:  imageregistry.operator.openshift.io/v1
    Fields Type:  FieldsV1
    fieldsV1:
      f:metadata:
        f:finalizers:
          .:
          v:"imageregistry.operator.openshift.io/finalizer":
      f:spec:
        .:
        f:httpSecret:
        f:logging:
        f:managementState:
        f:proxy:
        f:replicas:
        f:requests:
          .:
          f:read:
            .:
            f:maxWaitInQueue:
          f:write:
            .:
            f:maxWaitInQueue:
        f:rolloutStrategy:
        f:storage:
      f:status:
        .:
        f:conditions:
        f:generations:
        f:observedGeneration:
        f:readyReplicas:
        f:storage:
          .:
          f:pvc:
            .:
            f:claim:
        f:storageManaged:
    Manager:         cluster-image-registry-operator
    Operation:       Update
    Time:            2020-08-06T16:58:13Z
  Resource Version:  627357
  Self Link:         /apis/imageregistry.operator.openshift.io/v1/configs/cluster
  UID:               25942b86-fc8c-4e97-92bd-ec63eb07eb44
Spec:
  Default Route:     true
  Http Secret:       3fe77a6e47e1296fd4517c71029dbdf71317340f8dc782aa8f90f770ddc0f538b73b5b2d1c208e79748ff43424878105a16676133ce388b3efe07c5d88eb8364
  Logging:           2
  Management State:  Managed
  Proxy:
  Replicas:  1
  Requests:
    Read:
      Max Wait In Queue:  0s
    Write:
      Max Wait In Queue:  0s
  Rollout Strategy:       RollingUpdate
  Storage:
    Pvc:
      Claim:  crc-image-registry-storage
Status:
  Conditions:
    Last Transition Time:  2020-07-12T05:38:55Z
    Reason:                AsExpected
    Status:                False
    Type:                  ImageRegistryCertificatesControllerDegraded
    Last Transition Time:  2020-07-12T05:38:55Z
    Reason:                AsExpected
    Status:                False
    Type:                  NodeCADaemonControllerDegraded
    Last Transition Time:  2020-07-12T05:38:56Z
    Reason:                AsExpected
    Status:                False
    Type:                  ImageConfigControllerDegraded
    Last Transition Time:  2020-07-12T05:56:03Z
    Reason:                PVC Exists
    Status:                True
    Type:                  StorageExists
    Last Transition Time:  2020-08-06T16:58:13Z
    Message:               The registry is ready
    Reason:                Ready
    Status:                True
    Type:                  Available
    Last Transition Time:  2020-08-06T16:58:13Z
    Message:               The registry is ready
    Reason:                Ready
    Status:                False
    Type:                  Progressing
    Last Transition Time:  2020-07-12T05:39:15Z
    Status:                False
    Type:                  Degraded
    Last Transition Time:  2020-07-12T05:39:15Z
    Status:                False
    Type:                  Removed
  Generations:
    Group:              apps
    Hash:
    Last Generation:    5
    Name:               image-registry
    Namespace:          openshift-image-registry
    Resource:           deployments
  Observed Generation:  4
  Ready Replicas:       0
  Storage:
    Pvc:
      Claim:        crc-image-registry-storage
  Storage Managed:  false
Events:             <none>
```

Reply:
Thanks, @jboxman. As you've pointed out before, the term "CRD" seems to get confused with "CR" a lot. 

There were only two occurrences in this context, and I've changed both instances to `custom resource (CR)`.